velox/common/encode/Base64.h

```diff
@@ -122,8 +95,18 @@ class Base64 {
       size_t src_len,
       char* dst,
       size_t dst_len,
-      const ReverseIndex& table,
-      bool include_pad);
+      const ReverseIndex& table);
+
+ public:
+  // Encoding base to be used.
+  constexpr static int kBase = 64;
```

Comment:
Does this need to be public? I think it can be private.
Same for kBase in the Base32 class.

Reply:
If kBase is made private, then static_asserts for validation of the lookup table won't be able to access the kBase, since it is happening outside the class.